handlers/update_resource.go

```diff
@@ -27,45 +24,49 @@ type updateResourceEntry struct {
 // Handle the update resource request
 func (d *updateResourceEntry) Handle(params operations.UpdateResourceParams) middleware.Responder {
 	id := params.ID
-	resource := d.persistableResourceFromParams(id, params.Payload)
+	newResource := datautils.NewResource(params.Payload.(map[string]interface{})).WithID(id)
 
-	if errors := d.validator.ValidateResource(resource); errors != nil {
+	if errors := d.validator.ValidateResource(newResource); errors != nil {
 		return operations.NewUpdateResourceUnprocessableEntity().
 			WithPayload(&models.ErrorResponse{Errors: *errors})
 	}
 
-	_, err := d.database.Read(id)
+	existingResource, err := d.database.Read(id)
 	if err != nil {
 		if err.Error() == "not found" {
 			return operations.NewRetrieveResourceNotFound()
 		}
 		panic(err)
 	}
 
-	if err = d.database.Update(resource); err != nil {
-		panic(err)
-	}
+	newResource = datautils.NewResource(d.mergeJSON(&existingResource.JSON, &newResource.JSON)).WithID(id)
 
-	if err = d.addToStream(id); err != nil {
```

Comment:
Are you intentionally removing this call to `addToStream`? That seems unrelated to the stated purpose of this PR.

Reply:
Yes, intentionally as we're removing it as needed. I can back that change out and include it in a follow on PR later though if that's more comfortable. I was in the midst of editing this when we made that call, so I (perhaps aggressively) removed it...

Reply:
It seems like if you're going to remove it you'd also want to remove https://github.com/sul-dlss-labs/taco/blob/cd4753007d2eb3e40ce8e27586f7e67f1d3dd5e0/handlers/update_resource.go#L20 and update the NewUpdateResource not to take stream as an argument as well as update the `handlers/server.go` not to pass the argument, etc.  In short, yeah, I think reverting this change here is the easier path.